src/redux/configureStore.js

```diff
@@ -1,12 +1,32 @@
-import { combineReducers, configureStore } from '@reduxjs/toolkit';
-import reducerBooks from './books/books';
+import { combineReducers, configureStore, applyMiddleware } from '@reduxjs/toolkit';
+import thunk from 'redux-thunk';
+import reducerBooks, { fetchBooksSuccess } from './books/books';
 import reducerStatus from './categories/categories';
 
+const appID = 'RytECUH07JHLCqOksOjA';
+const baseUrl = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appID}/books`;
+
 const rootReducer = combineReducers({
   books: reducerBooks,
   status: reducerStatus,
 });
 
-const store = configureStore({ reducer: rootReducer });
+const fetchBooks = () =>
+  function (dispatch) {
+    fetch(baseUrl)
+      .then((response) => response.json())
+      .then(data => {
+        const books = data;
+        dispatch(fetchBooksSuccess(books));
+      });
+  };
```

Comment:
- [ ] Good job so far 👍 It would be great if you could remove all code relating to fetch in the `configureStore.js` and place it inside the `book.js` file 👍 
 - [ ] Also, there is a bug when adding successive books and removing them as you'll see in the video below. Kindly fix that issue 💯 
 - Your site:
![Animation](https://user-images.githubusercontent.com/85113567/174226987-bae6cbbf-87b4-47fa-a4ea-4ffc1f319601.gif)